pkgs/development/tools/misc/texinfo/6.0.nix

```diff
@@ -9,8 +9,11 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ perl xz ]
+    ++ stdenv.lib.optional stdenv.isSunOS libiconv
     ++ stdenv.lib.optional interactive ncurses;
 
+  configureFlags = "AWK=${gawk}/bin/awk";
```

Comment:
Yes it's a hidden dependency I've discovered while building on SmartOS.
Only GNU's awk works, SmartOS ships with `nawk` instead.